esx_service/utils/auth_data_test.py

```diff
@@ -0,0 +1,286 @@
+        self.assertTrue(uuid.UUID(tenant1.id))
+        
+        vms = []
+        privileges = []
+        tenant2 = self.auth_mgr.create_tenant('tenant2', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant2.id))
+      
+    def test_add_vms_to_tenant(self): 
+        vms = []
+        privileges = []
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+            
+        vms = [(str(uuid.uuid4()), 'vm1'), (str(uuid.uuid4()), 'vm2')]
+        tenant1.add_vms_to_tenant(self.auth_mgr.conn, vms)
+    
+    def test_remove_vms_from_tenant(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        
+        privileges = self.get_privileges()
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+
+        vms = [(str(uuid.uuid4()), 'vm2')]
+        tenant2 = self.auth_mgr.create_tenant('tenant2', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant2.id))
+
+        vms = [(str(uuid.uuid4()), 'vm3'), (str(uuid.uuid4()), 'vm4')]
+        tenant3 = self.auth_mgr.create_tenant('tenant3', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant3.id))
+
+        tenant3.remove_vms_from_tenant(self.auth_mgr.conn, vms)
+    
+    def test_set_name(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        
+        privileges = self.get_privileges()
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+
+        tenant1.set_name(self.auth_mgr.conn, 'new_tenant1')
+
+    
+    def test_set_description(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        
+        privileges = self.get_privileges()
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+        tenant1.set_description(self.auth_mgr.conn, 'new description')
+    
```

Comment:
this question applies to all tests -after something is being set, I'd expect the code to get the same stuff and compare the result with what was set. I do not see this code. What am I missing ?